classes.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class DbConnect
+{
+    private string $host = "127.0.0.1:3306";
+    private string $user = "root";
+    private string $pwd = "password";
+    private string $dbName = "collectorapp";
+
+    protected function connect(): PDO
+    {
+        $dsn = 'mysql:host=' . $this->host . ';dbname=' . $this->dbName;
+        $pdo = new PDO($dsn, $this->user, $this->pwd);
+        $pdo->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+        return $pdo;
+    }
+}
+
+class Bottles extends DbConnect
```

Comment:
it's generally bad practice to have multiple classes in one file, but to have them in separate files you have to use some other fancy features we haven't learned about, which is part of why they weren't in this project lol